admin-app/src/shared/chat-list/components/settings/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+export const SettingsSvg = () => {
+  return (
+    <svg
```

Comment:
svg must be in files with the extension .svg

Reply:
I wanted pass props here to change color of the setting svg. Is it works when we import svg as file.svg?

Reply:
use css, like .svg {path { stroke: #FFFFFF }}